ibazel/BUILD

```diff
@@ -20,7 +20,11 @@ go_binary(
     importpath = "github.com/bazelbuild/bazel-watcher/ibazel",
     # Since we don't have any cgo dependencies, force build to be pure
     # golang. This will make multi-os compatibility *MUCH* easier.
-    pure = "on",
+    pure = "auto",
```

Comment:
@achew22 I'm no go expert, so I do not really no the ramifications of toggling this, but I needed to do so in order to 
pull in FSEvents since it has C components

Reply:
This is an unfortunate thing to have to change. Turning pure off means that the binaries are no longer statically linked and we are using the cgo runtime (much slower, much buggier, much more pain) on mac.

Is this because of the dependency on fsevents?

If we decide to go this route we should definitely modify the above comment to reflect current state.

Reply:
Oh man, I didn't realize it would change the runtime, I thought it only affected how it was compiled and assembled. The FSEvents packages does have [`C` imports](https://github.com/fsnotify/fsevents/blob/master/wrap.go#L6-L26) that link specifically with the macOS `CoreServices.framework` which includes the bindings for FSEvents. It *seemed* that I was required to make the go binary with `cgo` and `pure: off` due to this constraint, but perhaps there is another workaround I am not aware of? Any ideas?


Side note; I was unable to make `pure` configurable with a select. I think its value is used in the macro to change things. Instead of `auto` I could duplicate the `go_binary` for macOS and set `pure: off` and leave the other as `pure: on` and could configure an alias for the proper platform. Would you prefer that?

Reply:
> Side note; I was unable to make pure configurable with a select. I think its value is used in the macro to change things. Instead of auto I could duplicate the go_binary for macOS and set pure: off and leave the other as pure: on and could configure an alias for the proper platform. Would you prefer that?

Oh `select`... your limitations are so impossible to understand...

I think I'm okay with this since auto will infer it from command line flags. It's too bad that we will have to do a cgo binary, but I think that's the price of having a good integration with osx.